src/doc/using_visit/MakingItPretty/Color_tables.rst

```diff
@@ -523,16 +520,20 @@ Tags must now only be comprised of alphanumeric characters, whitespace, dashes,
 Searching for Color Tables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Another option for finding the right color table is to enable searching. 
-With this option, users can type a search term into the **Name** text box, and only color tables that have a name containing that term will appear in the color table name table.
+Another option for finding the right color table is to use the search bar.
+With this option, users can type a search term into the **Search** text box, and only color tables that have a name containing that term will appear in the color table name table.
```

Comment:
Does the search feature match only text in name? Or, does it also match text in tags? Is the matching case-sensitive?

My prefs...case insensitve matching of either the name of the color table or the tag(s) associated with it.

Do we support any remarks associated with a color table? Like, `"This color table is designed for faciliating data differencing."` If so, maybe matching that text via search as well.

Reply:
> Does the search feature match only text in name?

The search feature only matches text in the color table names.

> Or, does it also match text in tags?

We do not support searching for color tables that have tags that match the search.

> Is the matching case-sensitive?

The matching is case-insensitive.

Reply:
Works for me.

Reply:
> My prefs...case insensitve matching of either the name of the color table or the tag(s) associated with it.

I think matching on the tags could get confusing. The search feature is designed to be used in tandem with the tag filtering. So, you can choose your favorite tags and then use search to whittle down the results more, or the opposite: you could start with a search and then use tags to get that list down to the results you want. I think having them behave independently of one another makes the system a bit easier to use.

Reply:
> Do we support any remarks associated with a color table?

Right now we don't support remarks for color tables. I think we talked about adding that in the past. I'll add that idea to the running list I have #18136 18136